app/controllers/foods_controller.rb

```diff
@@ -20,10 +20,10 @@ def edit; end
   # POST /foods or /foods.json
```

Comment:
![giphy](https://user-images.githubusercontent.com/93735137/168593977-8be1aa06-5695-49c9-9784-0d477ffd63b3.gif)
# Status: Approved :tada: :tada: :tada: 
Hi @abdessalem1998    :wave: :wave: 

![200w](https://user-images.githubusercontent.com/75126481/151663786-7e0aa330-6769-4ab2-a051-3754be515be9.gif)  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._